laravel/app/Models/Attending.php

```diff
@@ -18,7 +18,7 @@ class Attending extends Model
      *
      * @var array
      */
-    protected $guarded = ["id"];
+    protected $guarded = [];
```

Comment:
why are we "un-guarding" the `id`?

Reply:
The id has to be manually set - the id in the attending table is the medhub id for that attending. I left the resident id guarded because it is just an id (that doesn't hold any other meaning) and has auto incrementing

Reply:
Do you mind pointing out the line number(s) where setting the id happens? This is a big PR and I can't seem to find it.

Reply:
We make a request to MedHub for the attending's medhub id. If we actually get a medhub id, we set the Attending on lines 309-317.

Reply:
lines 309-317 of what file...?